### PR TITLE
Avoid panic in ReadStringFromSecretManager in secretmanager.go

### PR DESCRIPTION
**Pull request by pablodeymo**
Fixes # .

### What Changed?
Avoid panicking in secretmanager.go, returning the error instead of doing a `Fatal`.

### Reviewer Checklist
- [ ] Code is well-documented
- [ ] Code adheres to Go [naming conventions](https://go.dev/doc/effective_go#names)
- [ ] Code deprecates any old functionality before removing it

### CODE REVIEW

File: aws/secretmanager/secretmanager.go [hunk 1]
```diff
@@ -12,7 +11,7 @@ import (
 func ReadStringFromSecretManager(ctx context.Context, secretName, region string) (string, error) {
 	config, err := config.LoadDefaultConfig(ctx, config.WithRegion(region))
 	if err != nil {
-		log.Fatal(err)
+		return "", err
```
Comment (samlaf) [inline aws/secretmanager/secretmanager.go]:
```suggestion
		return "", errors.Wrap("error while loading default config", err)
```

Comment (samlaf) [reply aws/secretmanager/secretmanager.go]: nit: we have a wrap errors function somewhere in utils. Would you mind wrapping with that to give error context?

**Review by samlaf — COMMENTED** [review]
Agree with this, would just prefer errors to be wrapped everywhere.

File: aws/secretmanager/secretmanager.go (hunk 1 above)
Comment (pablodeymo) [reply aws/secretmanager/secretmanager.go]: done! thanks

**Review by shrimalmadhur — APPROVED** [review]

**Pull request merged**